backend/sozisel/lib/sozisel/model/events.ex

```diff
@@ -49,4 +49,16 @@ defmodule Sozisel.Model.Events do
   def change_event(%Event{} = event, attrs \\ %{}) do
     Event.create_changeset(event, attrs)
   end
+
+  def marshal_participant_event_data(%{__struct__: Quiz} = event_data) do
+    quiz_questions_data =
+      event_data.quiz_questions
+      |> Enum.map(&Map.take(&1, [:question_id, :question, :answer]))
+
+    event_data
+    |> Map.take([:duration_time_sec, :tracking_mode])
+    |> Map.put(:quiz_questions, quiz_questions_data)
+  end
+
+  def marshal_participant_event_data(other), do: other
```

Comment:
You can for now delete the other pattern matching as we probably will want to crash it if we don't support it yet I guess